samples/maps/demo/animated-map/demo.css

```diff
@@ -0,0 +1,22 @@
+#container {
+    min-width: 320px;
+    max-width: 800px;
+    height: 500px;
+    margin: 1em auto;
+}
+
+.animatedLine {
+    stroke-dasharray: 8;
+    stroke-dashoffset: 10;
+    animation: dash 5s linear infinite;
```

Comment:
There's a jaggy movement of the animation every 5s when the animation starts over. Are we able to tune this to make a smooth movement?